eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -224,10 +227,20 @@ stages:
                           SkipDefaultCheckout: true
                           Paths:
                             - sdk
+                            - sdk/**/*.md
```

Comment:
Is this necessary? I thought `sdk` would grab these files.

Reply:
Yes, redundant